src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -35,118 +39,182 @@
 import org.bukkit.util.Vector;
 
 @Name("Yaw / Pitch")
-@Description("The yaw or pitch of a location or vector.")
+@Description({"The yaw or pitch of a location or vector.",
+		"A yaw of 0 or 360 represents the positive x direction. Adding a positive number to the yaw of a player will rotate it counter-clockwise.",
+		"A pitch of 90 represents the positive y direction, or upward facing. A pitch of -90 represents downward facing. Adding a positive number to the pitch will rotate the player upwards."})
 @Examples({"log \"%player%: %location of player%, %player's yaw%, %player's pitch%\" to \"playerlocs.log\"",
-	"set {_yaw} to yaw of player",
-	"set {_p} to pitch of target entity"})
-@Since("2.0, 2.2-dev28 (vector yaw/pitch)")
+		"set {_yaw} to yaw of player",
+		"set {_p} to pitch of target entity",
+		"add 90 to yaw of player # Rotates the player counter-clockwise"})
+@Since("2.0, 2.2-dev28 (vector yaw/pitch), INSERT VERSION (changers)")
 public class ExprYawPitch extends SimplePropertyExpression<Object, Number> {
 
 	static {
-		register(ExprYawPitch.class, Number.class, "(0¦yaw|1¦pitch)", "locations/vectors");
+		register(ExprYawPitch.class, Number.class, "(:yaw|pitch)", "entities/locations/vectors");
 	}
 
+	// For non-Paper versions lower than 1.19, changing the rotation of an entity is not supported for players.
+	private static final boolean SUPPORTS_PLAYERS = Skript.isRunningMinecraft(1, 19) && Skript.getServerPlatform() == ServerPlatform.BUKKIT_PAPER;
+
 	private boolean usesYaw;
 
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		usesYaw = parseResult.mark == 0;
-		return super.init(exprs, matchedPattern, isDelayed, parseResult);
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```

Comment:
It's better to keep this same as super and other codes
```suggestion
	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```